qt-models/filtermodels.cpp

```diff
@@ -75,7 +75,7 @@ bool MultiFilterSortModel::showDive(const struct dive *d) const
 		bool found_tag = false;
 		for (const auto& filter_tag : m_filterData.tags) {
 			for (const auto& dive_tag : dive_tags) {
-				if (filter_tag.trimmed() == dive_tag.trimmed()) {
+				if (dive_tag.trimmed().toUpper().contains(filter_tag.trimmed().toUpper())) {
```

Comment:
and again, you introduced this two commits ago.
your commit series is more a "train of thought" exercise :-)

Reply:
That's a true statement.